react-todo/src/TodoList.jsx

```diff
@@ -0,0 +1,23 @@
+const todolist = [
+    {id: 1, title:"Brain storm ideas for to do app"},
+    {id: 2, title: "Create a list"},
+    {id: 3, title: "Complete assignment"}
+  ];
+  
+
+
+
+
+function TodoList(){
+    return (
+        <div>
+          <h1>Todo List</h1>
```

Comment:
Would recommend you move the `<h1>` outside this component back into the App component, since this heading isn't directly tied to the list itself.